frontend/projects/upgrade/src/styles.scss

```diff
@@ -253,6 +257,23 @@ button[mat-icon-button] {
   width: 100%;
 }
 
+.form-standard {
+  display: flex;
+  flex-direction: column;
+  row-gap: 16px;
+  padding: 32px 32px 16px;
+}
+
+.form-hint {
+  color: var(--dark-grey);
+}
+
+.learn-more-link {
+  color: var(--blue);
+  text-decoration: underline;
```

Comment:
in figma we don't have underlines in learn more redirect links, can you confirm @zackcl ?